### ISSUE TITLE
FR:  Add "Send" target to choose pre-FX, post-FX, and post fader

### ISSUE THREAD

**Issue by RattlesnakeLodge** [helgoboss/realearn#755]
It is possible to add a three-value target for sends?  The three values are currently only accessible via REAPER's own GUI.  There is not even a built-in REAPER action for this, which I can tie in to ReaLearn.

Thanks in advance!

![image](https://user-images.githubusercontent.com/106298585/201208125-861ef102-5d73-461b-97cd-d00522b5d980.png)

**Comment by PucksEvilTwin** [comment]
Yes, it's nice to see what type of send you are dealing with.  I have a spot dedicated to that in my layout already.
